src/dmd/cparse.d

```diff
@@ -2402,6 +2437,26 @@ final class CParser(AST) : Parser!AST
         }
     }
 
+    /*************************
+     * Simple __asm__ parser
+     * https://gcc.gnu.org/onlinedocs/gcc/Asm-Labels.html
+     * simple-asm-expr:
+     *   asm ( asm-string-literal )
+     *
+     * asm-string-literal:
+     *   string-literal
+     */
+    private AST.Expression cparseAsmLabel()
```

Comment:
Why isn't this named `cparseSimpleAsmExpr` ?
Also, it needs to be accounted for in the lookahead parser.

Reply:
#12558